TelepresenceBot/core/src/main/java/com/novoda/tpbot/human/HumanPresenter.java

```diff
@@ -23,8 +23,13 @@ void startPresenting(String serverAddress) {
     }
 
     void stopPresenting() {
-        observable.detachObservers();
-        humanTpService.disconnect();
+        if (observable != null) {
```

Comment:
In this case probably not. This presenter is not bound to only the `start` and `stop` of an `activity` but to a `ViewSwitcher`.